chia/wallet/wallet_transaction_store.py

```diff
@@ -52,25 +52,23 @@ async def create(cls, db_wrapper: DBWrapper):
             "CREATE INDEX IF NOT EXISTS tx_confirmed_index on transaction_record(confirmed_at_height)"
         )
 
-        await self.db_connection.execute(
-            "CREATE INDEX IF NOT EXISTS tx_created_index on transaction_record(created_at_time)"
-        )
-
+        # index tx_created_index is a duplicate to tx_created_time and can be dropped
+        await self.db_connection.execute("DROP INDEX IF EXISTS tx_created_index on transaction_record(created_at_time)")
```

Comment:
is there a good reason to change the formatting here? I would expect `flake8` or `black` to complain

Reply:
didn't get any `flake8` or `black` error on this. 
I can replace the CREATE INDEX by the DROP INDEX as well. 
I just added a comment and wanted the index to be dropped, as it is a 1:1 duplicate to tx_created_time.